bot/code_review_bot/report/phabricator.py

```diff
@@ -67,13 +67,12 @@ def publish(self, issues, revision, task_failures):
         if issues or task_failures:
             # Split issues and build errors
             issues_only = [issue for issue in issues if not issue.is_build_error()]
```

Comment:
You can remove that and use issues everywhere

Reply:
sure